protected/humhub/widgets/CoreJsConfig.php

```diff
@@ -61,8 +62,8 @@ public function run()
                     ],
                     'text' => [
                         'error.upload' => Yii::t('base', 'Some files could not be uploaded:'),
-                        'error.unknown' => Yii::$app->user->isAdmin() ?
-                            Yii::t('base', 'An unknown error occurred while uploading. Hint: check your upload_max_filesize and post_max_size php settings.') : Yii::t('base', 'An unknown error occurred while uploading.'),
+                        'error.unknown' => Yii::t('base', 'An unknown error occurred while downloading. Check if your file weight is less than {maxUploadSize}.', ['maxUploadSize' => Yii::$app->formatter->asShortSize((new FileValidator)->getSizeLimit())]) . (Yii::$app->user->isAdmin() ?
```

Comment:
@Semir1212 Can you please review this message? Maybe something link this: `An unknown error occurred while uploading the file. (Possibly the server upload limit of 5 GB was exceeded.)`

Reply:
@luke An unexpected error occurred. Please check whether your file exceeds the allowed upload limit of {maxUploadSize}.